### DISCUSSION TITLE
So amazing!

### DISCUSSION THREAD

**Discussion by ghost** [skills/.github#118] [General]
It was an amazing experience! I wonder if there is more to it...🤔

**Comment by Sithninja68** [comment]
Good try my friend

On Sun, Nov 20, 2022 at 12:59 AM Abhinav Singh ***@***.***>
wrote:

> It was an amazing experience! I wonder if there is more to it...🤔
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/skills/discussions/118>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZ6SLWG6QQ45XXR4E4TZXT3WJG43ZANCNFSM6AAAAAASFVUGGY>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
-- 
Sent from Gmail Mobile
